### DISCUSSION TITLE
Github Profile Page Public Email NOT Showing!

### DISCUSSION THREAD

**Discussion by Sang-Buster** [community/community#36478] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, 

My GitHub profile page does not show my public email when I signed out of my account, it only shows it when I'm logged in. When I am logged in, no other devices can see my public email either. I used other devices to check it, still don't see it, so I don't think it's a browser cache issue. 

I have configured the `Public Email` to my email. Under `Email Settings`, I've also added my email to my email list, verified it, and set it as my primary email. `Keep my email addresses private` option is also unchecked. So, I am clueless about how to solve this issue.

Anyone's help will be appreciated!
Thanks!!!!!

**Comment by shirinapr** [comment]
have you verified your email?

**Reply by Sang-Buster** [reply]
Yes, indeed

**Reply by Sang-Buster** [reply]
I even tried with my other email as my primary and public emails, still doesn't take effect.

**Comment by shirinapr** [comment]
go to [settings](https://github.com/settings/emails) read "Keep my email addresses private". 
add your github email in your git config.

**Reply by Sang-Buster** [reply]
I have also done that, my `git config user.name` and `git config --global user.name` are the same as of my public email (ง ͠ಥ_ಥ)ง

**Reply by ARMMWORXS** [reply]
Thanks.  JAR.

**Comment by MeesJ** [comment, marked as answer]
Your email address will only show for people who are signed into their GitHub account, not for guests (GitHub users not logged in).

**Reply by Sang-Buster** [reply]
I see, thank you! I wish GitHub gives us this option in the future, and let the guests see your email 😔 Thanks again :)

**Reply by MeesJ** [reply]
Agreed and no problem :)

**Comment by ossanna16** [comment]
Thank you for asking a great question @Sang-Buster! Glad you found our community 👍

**Comment by Rajeshkannan-Muthukumar** [comment]
I am currently facing the same problem , how you resolved it, please tell me, Sang-Butser

**Comment by Rajeshkannan-Muthukumar** [comment]
Hi,

My GitHub profile page does not show my public email when I signed out of my account, it only shows it when I'm logged in. When I am logged in, no other devices can see my public email either. I used other devices to check it, still don't see it, so I don't think it's a browser cache issue.
